src/components/Textarea.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+const DEFAULT_WIDTH = "300px";
+
+const Textarea = ({ name, width = DEFAULT_WIDTH, handleChange }) => {
+  return (
+    <div
+      style={{
+        display: "flex",
+        flexDirection: "column",
+        width,
+        justifyContent: "start"
+      }}
+    >
+      <textarea id={name} onChange={handleChange} />
```

Comment:
@pasternakewa is his controlled or uncontrolled component, don't see value to display here.

Reply:
```
      <Textarea
        handleChange={handleFormStateChange("note")}
        value={formState.note}
      />
```
It is controlled from the Form component

Reply:
Thank you. Fixes has just been commited.